### PR TITLE
Improvements on the campaign pages

### PR DESCRIPTION
**Pull request by cschweikert**
- layout improvements on the different cards
- improved donation item
- donation item in details

Please note: I'm not yet happy with how the table within the detail pages are looking, but I already have some ideas to make it look a bit more cool 😉

### CODE REVIEW

File: donation-tracker/src/modules/campaignDetails.tsx [hunk 1]
```diff
@@ -56,62 +57,65 @@ const CampaignDetailsModule = (props: any) => {
...
+                </Typography>
+                {donationItems
+                  .map((donationItem, index) => ({ donationItem, index }))
+                  .map(props => (
+                    <Box key={props.index} display="flex" flexDirection="row">
+                      <DonationRow {...props}></DonationRow>
+                      <Typography px={2} mt={'4px'} flex="1 0 40%" textAlign="right">
+                        {props.donationItem.remainingNeed
+                          ? `${props.donationItem.remainingNeed} ${props.donationItem.unit} still needed / werden noch gebraucht`
+                          : 'Goal accomplished/ Ziel erreicht'}
```
Comment (ManuelFeller) [inline donation-tracker/src/modules/campaignDetails.tsx]: cannot add the space before / on mobile, but for consistency I would go for it

**Review by ManuelFeller — APPROVED** [review]
🙏👍

File: donation-tracker/src/modules/campaignDetails.tsx (hunk 1 above)
Comment (cschweikert) [reply donation-tracker/src/modules/campaignDetails.tsx]:
```suggestion
                          : 'Goal accomplished / Ziel erreicht'}
```

Comment (cschweikert) [reply donation-tracker/src/modules/campaignDetails.tsx]:
```suggestion
                          : 'Goal accomplished / Ziel erreicht'}
```

**Pull request merged**